src/Shell/Task/ExtractTask.php

```diff
@@ -122,7 +122,8 @@ protected function _getPaths()
             $response = $this->in($message, null, $defaultPath);
             if (strtoupper($response) === 'Q') {
                 $this->err('Extract Aborted');
-                return $this->_stop();
+                $this->_stop();
+                return;
```

Comment:
One the bright side the sniff did catch few valid errors too in `@return` tag.

In order to address the errors reported by `PSR1.Files.SideEffects.FoundWithSymbols` can the `Router::reload();` and `Inflector::reset();` calls be moved from the class files to core or app's bootstrap?